RecFCCeeCalorimeter/src/components/AugmentClustersFCCee.h

```diff
@@ -80,6 +88,18 @@ class AugmentClustersFCCee : public Gaudi::Algorithm
   /// Name of the layer/cell field
   Gaudi::Property<std::vector<std::string>> m_layerFieldNames{
       this, "layerFieldNames", {"layer"}, "Identifiers of layers, corresponding to systemIDs"};
+  Gaudi::Property<std::vector<std::string>> m_thetaFieldNames{
+      this, "thetaFieldNames", {"theta"}, "Identifiers of theta, corresponding to systemIDs"};
+  Gaudi::Property<std::vector<std::string>> m_moduleFieldNames{
+      this, "moduleFieldNames", {"module"}, "Identifiers of module, corresponding to systemIDs"};
+  Gaudi::Property<bool> m_do_pi0_photon_shapeVar{
+      this, "do_pi0_photon_shapeVar", false, "Calculate shape variables for pi0/photon separation: E_ratio, Delta_E etc."};
```

Comment:
Hi @dasphy ,

does the code work for you if this property is set to false? In my local tests it crashes, can you please check?

Reply:
Hi @giovannimarchiori that works for me. In the RECO script I added "do_pi0_photon_shapeVar=True", to save the shape parameters:

   augmentCaloTopoClusters = AugmentClustersFCCee("augmentCaloTopoClusters",
                                                       inClusters=createTopoClusters.clusters.Path,
                                                       outClusters="Augmented" + createTopoClusters.clusters.Path,
                                                       systemIDs=[4],
                                                       systemNames=["EMB"],
                                                       numLayers=[ecalBarrelLayers],
                                                       readoutNames=[ecalBarrelReadoutName],
                                                       layerFieldNames=["layer"],
                                                       thetaRecalcWeights=[ecalBarrelThetaWeights],
                                                  do_pi0_photon_shapeVar=True,
                                                       OutputLevel=INFO)

Reply:
Hi @dasphy ,
yes, with do_pi0_photon_shapeVar=True the code runs, but as I wrote before, with do_pi0_photon_shapeVar=False, at least for me, there is a segmentation violation. Are you sure that for you with do_pi0_photon_shapeVar=False the code runs successfully?